### PR TITLE
Add text when no statistics found

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Adds a link to docs of statistics

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/entity/ha-statistic-picker.ts [hunk 1]
```diff
@@ -99,6 +65,56 @@ export class HaStatisticPicker extends LitElement {
...
+    <ha-svg-icon .path=${mdiCheck}></ha-svg-icon>
+    <paper-icon-item>
+      <state-badge slot="item-icon" .stateObj=${item.state}></state-badge>
+      <paper-item-body two-line="">
+        ${item.name}
+        <span secondary
+          >${item.id === "" || item.id === "__missing"
+            ? html`<a
+                target="_blank"
+                rel="noopener noreferer"
```
Comment (balloob) [inline src/components/entity/ha-statistic-picker.ts:104]:
```suggestion
                rel="noopener noreferrer"
```

File: src/components/entity/ha-statistic-picker.ts [hunk 2]
```diff
@@ -99,6 +65,56 @@ export class HaStatisticPicker extends LitElement {
...
+                href="${documentationUrl(
+                  this.hass,
+                  "/integrations/statistics/"
```
Comment (balloob) [inline src/components/entity/ha-statistic-picker.ts:107]:
```suggestion
                  "/more-info/energy/statistics"
```

(i'm opening PR to add this)

File: src/components/entity/ha-statistic-picker.ts (hunk 1 above)
Comment (bramkragten) [reply src/components/entity/ha-statistic-picker.ts:104]: 🙈 copy paste from yours in grid settings 😛

File: src/components/entity/ha-statistic-picker.ts [hunk 3]
```diff
@@ -99,6 +65,56 @@ export class HaStatisticPicker extends LitElement {
...
+      <paper-item-body two-line="">
+        ${item.name}
+        <span secondary
+          >${item.id === "" || item.id === "__missing"
+            ? html`<a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="${documentationUrl(
+                  this.hass,
+                  "/more-info/energy/statistics"
```
Comment (balloob) [inline src/components/entity/ha-statistic-picker.ts:107]:
```suggestion
                  "/more-info/statistics/"
```

**Pull request merged**